docs/topics/overriding-rules.adoc

```diff
@@ -72,6 +74,8 @@ The following rule override example changes the `effort` of the `weblogic-02000`
 
 When you run {ProductShortName}, this rule overrides the original rule with the same rule ID. You can verify that the new rule was used by viewing the contents of the Rule Provider Executions Overview.
 
+In order for the override rules to run, `targetTechnology` within the override ruleset must match one of the targets selected by the user when the analysis is executed.
```

Comment:
I am not certain, but is the word 'executed' not problematic?

Reply:
This phrase should not be here at all.  i put it elsewhere and rephrased.